Lib/pydoc_data/topics.py

```diff
@@ -15231,3 +15230,336 @@
+ 'test.support': 'library/test',
+ 'test.support.bytecode_helper': 'library/test',
+ 'test.support.import_helper': 'library/test',
+ 'test.support.os_helper': 'library/test',
+ 'test.support.script_helper': 'library/test',
+ 'test.support.socket_helper': 'library/test',
+ 'test.support.threading_helper': 'library/test',
+ 'test.support.warnings_helper': 'library/test',
+ 'textwrap': 'library/textwrap',
+ 'threading': 'library/threading',
+ 'time': 'library/time',
+ 'timeit': 'library/timeit',
+ 'tkinter': 'library/tkinter',
+ 'tkinter.colorchooser': 'library/tkinter.colorchooser',
+ 'tkinter.commondialog': 'library/dialog',
+ 'tkinter.dnd': 'library/tkinter.dnd',
+ 'tkinter.filedialog': 'library/dialog',
+ 'tkinter.font': 'library/tkinter.font',
+ 'tkinter.messagebox': 'library/tkinter.messagebox',
+ 'tkinter.scrolledtext': 'library/tkinter.scrolledtext',
+ 'tkinter.simpledialog': 'library/dialog',
+ 'tkinter.tix': 'library/tkinter.tix',
+ 'tkinter.ttk': 'library/tkinter.ttk',
+ 'token': 'library/token',
+ 'tokenize': 'library/tokenize',
+ 'trace': 'library/trace',
+ 'traceback': 'library/traceback',
+ 'tracemalloc': 'library/tracemalloc',
+ 'tty': 'library/tty',
+ 'turtle': 'library/turtle',
+ 'turtledemo': 'library/turtle',
+ 'types': 'library/types',
+ 'typing': 'library/typing',
+ 'unicodedata': 'library/unicodedata',
+ 'unittest': 'library/unittest',
+ 'unittest.mock': 'library/unittest.mock',
+ 'urllib': 'library/urllib',
+ 'urllib.error': 'library/urllib.error',
+ 'urllib.parse': 'library/urllib.parse',
+ 'urllib.request': 'library/urllib.request',
+ 'urllib.response': 'library/urllib.request',
+ 'urllib.robotparser': 'library/urllib.robotparser',
+ 'uu': 'library/uu',
+ 'uuid': 'library/uuid',
+ 'venv': 'library/venv',
+ 'warnings': 'library/warnings',
+ 'wave': 'library/wave',
+ 'weakref': 'library/weakref',
+ 'webbrowser': 'library/webbrowser',
+ 'winreg': 'library/winreg',
+ 'winsound': 'library/winsound',
+ 'wsgiref': 'library/wsgiref',
+ 'wsgiref.handlers': 'library/wsgiref',
+ 'wsgiref.headers': 'library/wsgiref',
+ 'wsgiref.simple_server': 'library/wsgiref',
+ 'wsgiref.util': 'library/wsgiref',
+ 'wsgiref.validate': 'library/wsgiref',
+ 'xdrlib': 'library/xdrlib',
+ 'xml': 'library/xml',
+ 'xml.dom': 'library/xml.dom',
+ 'xml.dom.minidom': 'library/xml.dom.minidom',
+ 'xml.dom.pulldom': 'library/xml.dom.pulldom',
+ 'xml.etree.ElementTree': 'library/xml.etree.elementtree',
+ 'xml.parsers.expat': 'library/pyexpat',
+ 'xml.parsers.expat.errors': 'library/pyexpat',
+ 'xml.parsers.expat.model': 'library/pyexpat',
+ 'xml.sax': 'library/xml.sax',
+ 'xml.sax.handler': 'library/xml.sax.handler',
+ 'xml.sax.saxutils': 'library/xml.sax.utils',
+ 'xml.sax.xmlreader': 'library/xml.sax.reader',
+ 'xmlrpc.client': 'library/xmlrpc.client',
+ 'xmlrpc.server': 'library/xmlrpc.server',
+ 'zipapp': 'library/zipapp',
+ 'zipfile': 'library/zipfile',
+ 'zipimport': 'library/zipimport',
+ 'zlib': 'library/zlib',
+ 'zoneinfo': 'library/zoneinfo'}
```

Comment:
In general, I would agree with Terry’s comment on b.p.o that it’s only necessary to store exceptions here, and use the previous path for simple module names.  On the other hand this is 300 lines in a huge file so…

Reply:
Right, I see now that storing a list of known names is the simplest way.
Instead of storing easily-derived paths or even True, what about having a list?

Reply:
I think at that point it's better to have the simplicity of consistency than the minor space savings.  At best, we could save about 5.7 of the 12.6 kB added here, in an 800 kB file